Assets/MixedRealityToolkit/Providers/Hands/HandRay/HandRay.cs

```diff
@@ -76,7 +78,6 @@ public void Update(Vector3 handPosition, Vector3 palmNormal, Transform headTrans
 
             stabilizedRay.AddSample(new Ray(measuredRayPosition, measuredDirection));
         }
-
```

Comment:
nit: we've typically left spaces around the region tags

Reply:
Fixed